Tests/configure_and_test_integration_instances.py

```diff
@@ -19,6 +19,7 @@
 from demisto_sdk.commands.validate.file_validator import FilesValidator
 from demisto_sdk.commands.common.constants import YML_INTEGRATION_REGEXES, INTEGRATION_REGEX, PACKS_INTEGRATION_REGEX, \
     BETA_INTEGRATION_REGEX, RUN_ALL_TESTS_FORMAT
+from distutils.version import LooseVersion
 from demisto_sdk.commands.common.tools import server_version_compare
```

Comment:
```suggestion
```
Need to delete this import if you're going to use the `LooseVersion` comparison instead.